zcash_primitives/CHANGELOG.md

```diff
@@ -90,6 +90,10 @@ and this library adheres to Rust's notion of
     `&self`.
 - `zcash_primitives::merkle_tree::CommitmentTree::new` has been renamed to
   `CommitmentTree::empty`.
+- `zcash_primitives::primitives::SaplingIvk` struct was added. This newtype wrapper is
```

Comment:
Sorry, I wasn't clear:
- "`zcash_primitives::primitives::SaplingIvk` struct." needs to go in the "Added" section (above the equivalent line for `Nullifier`), and this section then becomes:
```suggestion
- `zcash_primitives::primitives::SaplingIvk` is
```